extensions/notebook/src/jupyter/localCondaPackageManageProvider.ts

```diff
@@ -0,0 +1,72 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { IPackageManageProvider, IPackageDetails, IPackageTarget } from '../types';
+import { JupyterServerInstallation } from './jupyterServerInstallation';
+import * as constants from '../common/constants';
+
+export class LocalCondaPackageManageProvider implements IPackageManageProvider {
+
+
+	/**
+	 * Provider Id for Anaconda package manage provider
+	 */
+	public static ProviderId = 'localhost_Anaconda';
+
+	constructor(private jupyterInstallation: JupyterServerInstallation) {
```

Comment:
this just reminded me of something that we could noodle on. Should we have a way to get the jupyter server installation from anywhere within the notebook extension (because there should only be one)? I could be wrong but I feel like we need to pass it all over the place.

Reply:
passing this here gives me the option to add test for this class. which I will in next PR :)

---
In reply to: [350547675](https://github.com/microsoft/azuredatastudio/pull/8473#discussion_r350547675) [](ancestors = 350547675)